package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "logdna-agent",
   "version": "1.6.5",
-  "description": "LogDNA Collector Agent",
+  "description": "LogDNA Agent which streams from log files to your LogDNA account. Works with Linux, Windows, and macOS Servers",
```

Comment:
I'd remove `which` from this.  `LogDNA Agent streams...`

Reply:
@darinspivey, done!